src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataReorderSection.tsx

```diff
@@ -33,13 +34,27 @@ const ReleaseDataReorderSection = ({ releaseId, canUpdateRelease }: Props) => {
   const [reorderingFilters, setReorderingFilters] = useState<Subject>();
   const [reorderingIndicators, setReorderingIndicators] = useState<Subject>();
 
-  const handleSaveFilters = (updatedFilters: UpdateFiltersRequest) => {
+  const handleSaveFilters = (
```

Comment:
For async/await stuff, we should write our handlers accordingly e.g.
```ts
const handleSaveFilters = async (...) => {
  await adminTableBuilderService.updateFilters(...);
}
```

Similar applies for `handleSaveIndicators` below as well

Reply:
Thanks have made this change.